programs/tsctp.c

```diff
@@ -647,9 +647,17 @@ int main(int argc, char **argv)
 					continue;
 				}
 #ifdef _WIN32
-				tid = CreateThread(NULL, 0, &handle_connection, (void *)conn_sock, 0, NULL);
+				if ((tid = CreateThread(NULL, 0, &handle_connection, (void *)conn_sock, 0, NULL)) == NULL) {
+					perror("CreateThread");
```

Comment:
You need to use `GetLastError()` on Windows to get the error number if `CreateThread()` fails.